lib/task_listener.js

```diff
@@ -181,6 +182,18 @@ export default class TaskListener extends EventEmitter {
               stack: e.stack
           });
         }
+        let stats = {
+          uptime: this.host.billingCycleUptime(),
+          interval: this.host.billingCycleInterval()
+        };
+        let remainder = stats.interval - (stats.uptime % stats.interval);
+        if(remainder * this.runtime.taskQueue.slowdownDivisor > stats.interval) {
```

Comment:
Sure, wait 40 minutes!  No problem in that..

or you could do something like:
https://github.com/taskcluster/docker-worker/blob/master/test/integration/shutdown_test.js#L31
https://github.com/taskcluster/docker-worker/blob/master/test/integration/shutdown_test.js#L13